### PR TITLE
fix(kanban-board): "added team and sprint filter in kanbar board, als…

### PR DESCRIPTION
**Pull request by abhishek-mishra019**
…o made some ui changes"

Signed-off-by: Abhishek Mishra <am1426620@gmail.com>

### Functionality:
fixing kanban board and making ui changes

### Solution:
added team and sprint filter in kanban board, also made some ui changes

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
step 1. on the kanban board click on team filter to switch team.
step 2. click to sprint filter to view tasks of a particular sprint

### CODE REVIEW

File: src/app/body/kanban-board/kanban-board.component.html [hunk 1]
```diff
@@ -1,6 +1,37 @@
...
+          <div class="input-group">
+            <h5>Sprint No.</h5>
+            <input class="form-control" style="font-size: 11px" type="number" id="filterSprintNumberInput"
+              [(ngModel)]="filterSprintNumber" />
+            <button id="dropdownButton" class="btn" style="font-size: 10px" (click)="changeSprintNumber()">
+              Filter
+            </button>
+          </div>
+        </div>
+      </div>
```
Comment (viv9k) [inline src/app/body/kanban-board/kanban-board.component.html:30]: I think this should be in col

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

File: src/app/body/kanban-board/kanban-board.component.html (hunk 1 above)
Comment (abhishek-mishra019) [reply src/app/body/kanban-board/kanban-board.component.html]: i have done the required changes, please look into it

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**